requirements.txt

```diff
@@ -1,4 +1,5 @@
-OWSLib>=0.8.3,<0.19.0
+OWSLib>=0.8.3,<0.19.0; python_version <= '2' #conda: OWSLib>=0.8.3,<0.19.0 (only python=2)
+OWSLib>=0.8.3; python_version > '2' #conda: OWSLib>=0.19.0 (only python=3)
```

Comment:
This case was not covered to I had to code it in `gen_conda_requirements.py`, but I cannot wait for `compliance-checker` to drop Python 2.7!

Reply:
Same.  For me, wrangling with Unicode issues while supporting both Python 2 and 3 was quite the headache back in the day.